net/respondd/src/respondd.c

```diff
@@ -649,10 +645,7 @@ int main(int argc, char **argv) {
 			break;
 
 		case 'd':
-			if (chdir(optarg)) {
```

Comment:
Seems a bit arbitrary to me. At least, every directory should have a chance to provide a cache time.